src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
+
+public class Circle extends Figure {
+    private double radius;
+    private double area;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+        this.area = calculateArea();
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+        this.area = calculateArea();
```

Comment:
Don't use static methods in your solution. The 'calculateArea()' method should not be static and should be called when needed, not in the setter method.